lib/course/middleware/exceptions_middleware.rb

```diff
@@ -0,0 +1,18 @@
+module Course
+  module Middleware
+    class ExceptionsMiddleware
+      attr_reader :app
+
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        status, headers, body = app.call(env)
+        [status, headers, body]
+      rescue StandardError => _e
```

Comment:
1. There is no need to specify StandardError, because it is default for the rescue
2. `e` is unused
3. What will happen if something raised an exception of a higher level then StandardError
```
BasicObject
  Exception
    NoMemoryError
    ScriptError
      LoadError
      NotImplementedError
      SyntaxError
    SecurityError
    SignalException
      Interrupt
    StandardError            <--- default for "rescue" statements
      ArgumentError
        UncaughtThrowError
      EncodingError
      FiberError
      IOError
        EOFError
      IndexError
        KeyError
        StopIteration
          ClosedQueueError
      LocalJumpError
      NameError
        NoMethodError
      RangeError
        FloatDomainError
      RegexpError
      RuntimeError
      SocketError
      SystemCallError
      ThreadError
      TypeError
      ZeroDivisionError
    SystemExit
    SystemStackError
    fatal
```